kubernetes/cmsweb/services/reqmgr2ms-ruleCleaner.yaml

```diff
@@ -0,0 +1,143 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: ms-rulecleaner
+  namespace: dmwm
+spec:
+  selector:
+    app: ms-rulecleaner
+  ports:
+    - port: 8244
+      targetPort: 8244
+      name: ms-rulecleaner
+    - port: 18244
+      targetPort: 18244
+      name: ms-rulecleaner-mon
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: ms-rulecleaner
+  labels:
+    app: ms-rulecleaner
+  namespace: dmwm
+data:
+  setup-certs-and-run.sh: |
+    #!/bin/bash
+    cp /host/etc/grid-security/* /etc/grid-security
+    echo 'INFO Files in /etc/grid-security'
+    ls -lahZ /etc/grid-security
+    su  -c "cd /data && /data/run.sh" --preserve-environment _reqmgr2ms
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  labels:
+    app: ms-rulecleaner
+  name: ms-rulecleaner
+  namespace: dmwm
+spec:
+  selector:
+    matchLabels:
+      app: ms-rulecleaner
+  replicas: 1
```

Comment:
Todor, we usually use `1 #PROD N` syntax here where N means how many replicas you'll need on production cluster. If you foresee that your service may require more then one replicas please specify via this syntax the max amount of replicas you may need.

Reply:
Hi @vkuznet , Thanks for mentioning it. So far we do not foresee more than a single instance of this  service in prod as well as in preprod. I hope we may change this  value in the far  future in case  we decide to do so, but in a near future I do not see a way to make it run in more than a singe instance without significant changes in the code.